fcal.cc

```diff
@@ -8,41 +8,37 @@
 
 #include "G4PhysListFactory.hh"
 #include "G4SystemOfUnits.hh"
+#include "G4VisExecutive.hh"
 
 
-#ifdef G4VIS_USE
-  #include "G4VisExecutive.hh"
-#endif
-
-
-int main(int argc, char** argv) 
```

Comment:
After removing G4VIS option from CMakeLists, it doesn't compile.
I just include it all the time. It doesn't hurt